teamscale-gradle-plugin/src/main/kotlin/com/teamscale/config/TestwiseCoverageConfiguration.kt

```diff
@@ -3,11 +3,22 @@ package com.teamscale.config
 import com.teamscale.TestImpacted
 import com.teamscale.client.EReportFormat
 import org.gradle.api.Project
+import org.gradle.api.file.FileCollection
+import org.gradle.api.file.RegularFileProperty
 import java.io.File
 
 /** Configuration for the testwise coverage report. */
+@Suppress("MemberVisibilityCanBePrivate")
 open class TestwiseCoverageConfiguration(project: Project, task: TestImpacted) :
     ReportConfigurationBase(EReportFormat.TESTWISE_COVERAGE, project, task) {
+
+    /** The destination where the report should be written to/read from. */
+    var destination: RegularFileProperty = project.objects.fileProperty()
+
+    /** Allows to set the destination as a string, which is resolved as File internally. */
+    fun setDestination(destination: String) {
+        this.destination.set(project.objects.fileProperty().fileValue(File(destination)))
+    }
```

Comment:
Not sure if this is done before, but if not, shouldn't we somehow handle wrong destinations here? Like destination is a directory, not readable/writable?